autogen/docs/behaviorspace.md.mustache

```diff
@@ -696,9 +845,10 @@ will hopefully be apparent how to use the tags to specify different kind of
 experiments. The DTD specifies which tags are required and which are optional,
 which may be repeated and which may not, and so forth.
 
-When XML for experiment setups is included in a model file, it does not begin
-with any XML headers, because not the whole file is XML, only part of it. If you
-keep experiment setups in their own file, separate from the model file, then the
+If you want to create a setup file for NetLogo 6.3.0 and earlier versions for 
+which **Export** is not available you need to know that in a model file the XML for
+ experiment setups does not begin with any XML headers, because the whole file is XML.
```

Comment:
I don't think the whole file is XML, so maybe change the end of this sentence.

Reply:
should have been (now is) "because the not whole file
 is XML, only part of it."